ui/packages/docs/src/components/AoE/character_data.json

```diff
@@ -6611,6 +6611,130 @@
       }
     ]
   },
+  "travelerhydro": {
+    "normal": [
+      {
+        "ability": "Aether-N1",
+        "shape": "Box",
+        "center": "Player",
+        "boxX": 1.4,
+        "boxY": 2.2
+      },
+      {
+        "ability": "Aether-N2",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 0.6,
+        "radius": 1.7,
+        "fanAngle": 180
+      },
+      {
+        "ability": "Aether-N3",
+        "shape": "Box",
+        "center": "Player",
+        "offsetY": 0.4,
+        "boxX": 1.5,
+        "boxY": 2.2
+      },
+      {
+        "ability": "Aether-N4",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 0.6,
+        "radius": 1.7
+      },
+      {
+        "ability": "Aether-N5",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 0.6,
+        "radius": 1.75,
+        "fanAngle": 240
+      },
+      {
+        "ability": "Lumine-N1",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 1,
+        "radius": 1.6
+      },
+      {
+        "ability": "Lumine-N2",
+        "shape": "Box",
+        "center": "Player",
+        "boxX": 1.4,
+        "boxY": 2.2
+      },
+      {
+        "ability": "Lumine-N3",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 0.7,
+        "radius": 1.5
+      },
+      {
+        "ability": "Lumine-N4",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 0.7,
+        "radius": 1.5
+      },
+      {
+        "ability": "Lumine-N5",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 1,
+        "radius": 1.6
+      }
+    ],
+    "charge": [
+      {
+        "ability": "CA-1",
+        "shape": "Circle",
+        "center": "Player",
+        "radius": 2.2
+      },
+      {
+        "ability": "CA-2",
+        "shape": "Circle",
+        "center": "Player",
+        "radius": 2.2
+      }
+    ],
+    "skill": [
+      {
+        "ability": "E-Torrent-Surge",
+        "shape": "Box",
+        "center": "Player",
+        "boxX": 1.2,
+        "boxY": 15
+      },
+      {
+        "ability": "E-Spiritbreath-Thorn",
+        "shape": "Box",
+        "center": "Player",
+        "boxX": 1.2,
+        "boxY": 15
+      },
+      {
+        "ability": "E-Hold",
+        "shape": "Box",
+        "center": "PrimaryTarget",
+        "offsetY": -0.4,
+        "boxX": 0.3,
+        "boxY": 1.3
+      }
+    ],
+    "burst": [
+      {
+        "ability": "Q-Tick",
+        "shape": "Circle",
+        "center": "GlobalValue",
+        "radius": 0.15,
+        "notes": "Starts at Traveler position in that direction and moves at 1.5m (or 1.05m if Traveler C2+) per attack."
```

Comment:
Please change this to `Starts at Traveler position + 0.5m in that direction and moves at 1.5m (or 1.05m if Traveler C2+) per attack.`